mix.exs

```diff
@@ -1,29 +1,46 @@
-defmodule Tee.MixProject do
+defmodule Membrane.Element.Tee.MixProject do
   use Mix.Project
 
+  @version "0.1.0"
+  @github_url "https://github.com/membraneframework/membrane-element-tee"
+
   def project do
     [
-      app: :tee,
-      version: "0.1.0",
+      app: :membrane_element_tee,
+      version: @version,
       elixir: "~> 1.7",
       start_permanent: Mix.env() == :prod,
-      deps: deps()
+      source_url: @github_url,
```

Comment:
if you provide name in `mix.exs`'s project it will render nicely in docs. 

You can also provide other fields present in cores mix.exs: https://github.com/membraneframework/membrane-core/blob/master/mix.exs#L7